<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="DESKTOP-TJU5IQN" failures="1" tests="3" name="TestP" time="37.859" errors="0" timestamp="2024-10-21T11:35:54 IST">
  <testcase classname="script.Demo1" name="testA" time="1.954"/>
  <testcase classname="script.Demo2" name="testB" time="0.025">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at script.Demo2.testB(Demo2.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- testB -->
  <testcase classname="script.Demo3" name="testC" time="0.011"/>
  <testcase classname="generic.BaseTest" name="@AfterSuite generateReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite generateReport -->
  <testcase classname="generic.BaseTest" name="@BeforeSuite initReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite initReport -->
  <testcase classname="generic.BaseTest" name="@AfterSuite generateReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite generateReport -->
  <testcase classname="generic.BaseTest" name="@BeforeSuite initReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite initReport -->
</testsuite> <!-- TestP -->
